finality-aleph/src/import.rs

```diff
@@ -122,10 +103,14 @@ where
             justifications.and_then(|just| just.into_justification(ALEPH_ENGINE_ID))
         {
             let res = self.import_justification(hash, number, (ALEPH_ENGINE_ID, justification));
-            res.unwrap_or_else(|_err| {
-                imported_aux.bad_justification = true;
+            if let Err(ConsensusError::ClientImport(message)) = res {
+                if message == "Aleph can import only Aleph justifications." {
```

Comment:
It looks strange that we are marking bad_justification only if the error message is matched.

Reply:
I agree, but I don't have any other idea how we could distinguish this error and it should be distinguished. `ConsensusError::ClientImport` takes only string arguments.

Reply:
Ok, I added a new function called `send_justification`